src/SFA.DAS.EAS.Portal.Client.TestHarness/DependencyResolution/LoggerRegistry.cs

```diff
@@ -0,0 +1,13 @@
+using SFA.DAS.NLog.Logger;
+using StructureMap;
+
+namespace SFA.DAS.EAS.Portal.Client.TestHarness.DependencyResolution
+{
+    public class LoggerRegistry : Registry
+    {
+        public LoggerRegistry()
+        {
+            For<ILog>().Use(c => new NLogLogger(c.ParentType, null, null)).AlwaysUnique();
```

Comment:
Might be better to use a NullLogger or FakeLogger for tests.

Reply:
I find it useful to view the log from the test harness, to review the appropriateness of our logging.